### ISSUE TITLE
How to get all events or past events

### ISSUE THREAD

**Issue by mmjaeger** [matteosavio/eventbritelist#1]
Hello
I was wondering how I could get all events or maybe just past events?

Thanks

**Comment by matteosavio** [comment]
yeah, I could do an option "events" which defaults to "upcoming" but can also be set to "past" or "all".

**Comment by mmjaeger** [comment]
That would be great - thanks for considering it

**Comment by mmjaeger** [comment]
Tried the latest version with some issues though:
1. the fontawesome js screws up some existing buttons - FA 5.0 interfering if FA 4.7 is already running
2. show_description always returns false - this fixes the issue: if(strval($showDescription) == "false")
3. I was wondering why getting those events from eventbrite takes rather long ~ may 10 - 20 secs.
4. Where do I find the profile ID and what is it - it is not the organizer ID, right?
